src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -627,6 +627,18 @@ void SetGlobalPropertiesRequest::PrepareUIRequestVRHelpData(
   out_params[strings::vr_help] = (*app->vr_help());
 }
 
+void CacheAlreadySavedKeyboardProperty(
+    const std::string& keyboard_property,
+    const smart_objects::SmartObject* saved_keyboard_props,
+    smart_objects::SmartObject& cached_keyboard_props) {
+  if (saved_keyboard_props &&
+      saved_keyboard_props->keyExists(keyboard_property)) {
+    cached_keyboard_props[keyboard_property] =
+        static_cast<hmi_apis::Common_KeyboardLayout::eType>(
+            (*saved_keyboard_props)[keyboard_property].asInt());
```

Comment:
@ychernysheva this cast is redundant by two reasons:
1. It always casts all incoming parameters to KeyboardLayout enum (however it can be any enum)
2. SmartObject has an `operator =` implementation for int, so `asInt()` is enough